docs-src/faq.rst

```diff
@@ -8,8 +8,8 @@ What even is |product_name| and why should I care?
 |product_name| is a wrapper around commonly accessed parts of the Slack Platform. It provides basic mechanisms for
 using the Slack Web API from within your Python app.
 
-On the other hand, |product_name| does not provide access to the RTM or Events bot-building APIs, although we anticipate
-adding support for these in the future.
+On the other hand, |product_name| does not provide access to the Events bot-building API, although we anticipate adding
```

Comment:
Maybe it would be good to add a reference to the [official Events API package](https://github.com/slackapi/python-slack-events-api), now that its built?

Reply:
So besides the RTM being now supported, the Events API is now supported too?

Reply:
its not supported with this pacakge, but the `slackeventsapi` package is another official one that people might want to know about.

Reply:
https://github.com/slackapi/python-slack-events-api/